tutor/src/models/reference-book/ux.js

```diff
@@ -75,6 +75,9 @@ export default class BookUX {
     if (this.book && !cs) {
       cs = first(this.book.pages.byChapterSection.keys());
     }
+    if (this.tours && this.tours.tourRide) {
+      setTimeout(() => invoke(this, 'tours.tourRide.joyrideRef.calcPlacement'), 10);
```

Comment:
is this timeout waiting for react to re-render? could you invoke the calcPlacement from the componentDidUpdate?

Reply:
yes, it's waiting for react to re-render the titles, but also potentially for mathjaxj to run.  Basically it's just waiting for the page to "settle down" and for everything to be re-positioned.